### PR TITLE
Adds mutation for editing post

### PR DESCRIPTION
**Pull request by drewwilliams5280**
Minor edit for adding mutation.

### CODE REVIEW

File: spec/graphql/mutations/posts/edit_post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+module Mutations
+  module Posts
+    RSpec.describe EditPost, type: :request do
+      describe '.resolve' do
+        it 'edits a post' do
+          new_user = create(:user, id: 40)
+          new_post = create(:post, user_id: 40, content: "He's left and right handed")
+          require 'pry'; binding.pry
```
Comment (Alex-Desjardins) [inline spec/graphql/mutations/posts/edit_post_spec.rb]: Looks like a pry was left behind here.

**Review by Alex-Desjardins — CHANGES_REQUESTED** [review]
Looks good! There was a just a pry left behind in your test do thats why the CI wont build.

**Comment by ggerdsen** [comment]
> Looks good! There was a just a pry left behind in your test do thats why the CI wont build.

Good catch!

**Comment by Alex-Desjardins** [comment]
Looks great!

**Review by Alex-Desjardins — APPROVED** [review]

**Pull request merged**